lib/spack/docs/module_file_support.rst

```diff
@@ -654,8 +655,8 @@ The allowed values for the ``autoload`` statement are either ``none``,
      In the ``tcl`` section of the configuration file it is possible to use
      the ``prerequisites`` directive that accepts the same values as
      ``autoload``. It will produce module files that have a ``prereq``
-     statement, which can be used to autoload dependencies in some versions
-     of Environment Modules.
+     statement, which autoloads dependencies on Environment Modules when its
+     ``auto_handling`` configuration is enabled.
```

Comment:
Does it make sense to document here what version of environment-modules enables this by default?

Reply:
Yes. I am adding this.